requirements.txt

```diff
@@ -0,0 +1 @@
+mkdocs-material
```

Comment:
fix: The requirements.txt should also contain `"mkdocs-material[imaging]"`, since this is also a dependency.

Reply:
Added!